client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -323,12 +345,17 @@ const connectComponent = connect( ( state, ownProps ) => {
 		Number( ownProps.fields.podcasting_category_id );
 	const isPodcastingEnabled = podcastingCategoryId > 0;
 
+	const categories = getTermsForQueryIgnoringPage( state, siteId, 'category', {} );
+	const selectedCategory = categories && head( filter( categories, { ID: podcastingCategoryId } ) );
+	const podcastingFeedUrl = selectedCategory && selectedCategory.feed_url;
```

Comment:
Is the reason this processing is done here and not in a selector in order to support dynamic changing of the URL when the category changes before it has been saved?

Reply:
Mostly because `podcastingCategoryId` isn't in global state. It comes from the fields "magic" that's in settings and gets updated in real-time as the category is updated. The podcast category id that's in state currently isn't updated until the settings are saved. I'm not sure if it's more correct to update feed URL in real-time if it gets displayed outside of the settings page. I wouldn't personally expect it to update until I saved the settings (much like the category indicator).